### PR TITLE
Go back to collectd_version

### PR DESCRIPTION
**Pull request by jyaworski**
Someone forgot to rename everything....

### CODE REVIEW

**Comment by jyaworski** [comment]
Credit to @caldwell for the scoping portion of this PR.

File: Gemfile [hunk 1]
```diff
@@ -23,7 +23,7 @@ group :test do
   gem 'rspec-puppet-facts'
   gem 'rspec'
   gem 'puppet-blacksmith'
-  gem 'rubocop'
+  gem 'rubocop', '0.35.0'
```
Comment (igalic) [inline Gemfile]: i'd love if you submit this also to modulesync_config

Comment (jyaworski) [inline Gemfile]: Did you not see?

https://github.com/voxpupuli/modulesync_config/pull/59#event-516658545

**Comment by duritong** [comment]
And someone forgot to clean up the README.md when reverting. It's completely misleading now and the minimum_version parameter is completely useless. :(

**Comment by igalic** [comment]
@duritong can you patch up README.md for us?

**Pull request merged**